src/evict/evict_lru.c

```diff
@@ -711,22 +711,31 @@ __evict_update_work(WT_SESSION_IMPL *session)
      */
     bytes_max = conn->cache_size + 1;
     bytes_inuse = __wt_cache_bytes_inuse(cache);
-    if (__wt_evict_clean_needed(session, NULL))
+    if (__wt_evict_clean_needed(session, NULL)) {
         LF_SET(WT_EVICT_CACHE_CLEAN | WT_EVICT_CACHE_CLEAN_HARD);
-    else if (bytes_inuse > (target * bytes_max) / 100)
+        WT_STAT_CONN_INCR(session, cache_eviction_trigger_clean_reached);
+    } else if (bytes_inuse > (target * bytes_max) / 100) {
         LF_SET(WT_EVICT_CACHE_CLEAN);
+        WT_STAT_CONN_INCR(session, cache_eviction_trigger_clean_reached);
```

Comment:
The `else` check is for the `target` not `trigger`. The `target` can reach many times, and it is expected, so we do not need to track it.